components/Dashboard/Cards/DashboardCard.js

```diff
@@ -1,31 +1,87 @@
 import PropTypes from 'prop-types';
 import { Box, Typography, Grid, Hidden } from '@material-ui/core';
-import { useTheme } from '@material-ui/core/styles';
+import { useTheme, makeStyles } from '@material-ui/core/styles';
 import Card from '../../Shared/Card';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    backgroundColor: theme.palette.primary.dark,
+    height: '100%',
+    width: '100%',
+    flexWrap: 'nowrap',
+    padding: '2rem',
+  },
+  iconButton: {
+    color: theme.palette.primary.contrastText,
+    marginLeft: 'auto',
+  },
+  button: {
+    width: '100%',
+    fontSize: '18px',
+    '&:focus': {
+      outline: 'none',
+    },
+    textTransform: 'capitalize',
+    color: '#BDACD4',
+    '&:hover': {
+      backgroundColor: '#977BBF',
+      color: '#fff',
+    },
+  },
+  friendsSection: {
+    flex: '1',
+    flexWrap: 'nowrap',
+    overflowY: 'auto',
+    overflowX: 'hidden',
+    padding: ({ open }) => open && theme.spacing(0, 1.5),
+    '& > ul': {
+      flexGrow: '1',
+    },
+    /* Hide scrollbar for Chrome, Safari and Opera */
+    '&::-webkit-scrollbar': {
+      display: ({ isSmallScreen }) => !isSmallScreen && 'none',
+    },
+    '-ms-overflow-style': 'none' /* IE and Edge */,
+    scrollbarWidth: ({ isSmallScreen }) => !isSmallScreen && 'none' /* Firefox */,
+  },
+}));
 const DashboardCard = ({ spaceName, description, variant }) => {
   const theme = useTheme();
+  const classes = useStyles();
+
   return (
     <Card>
       <Grid
-        spacing={5}
         container
         direction="row"
-        className="rounded-lg"
         style={{
           background: variant === 'dark' ? theme.palette.primary.mainGradient : theme.palette.secondary.mainGradient,
-        }}>
-        <Hidden mdDown>
-          <Grid item xs={4}>
-            <Box bgcolor="text.disabled" width={1} height={1}></Box>
+        }}
+        className={classes.container}>
+        <Hidden smDown>
+          <Grid item sm={4}>
+            <Box display="flex" alignItems="center" justifyContent="center" height={1} width={1}>
+              <Box bgcolor="text.disabled" height="5rem" width="5rem"></Box>
+            </Box>
           </Grid>
         </Hidden>
 
-        <Grid item xs={8}>
-          <Box color={variant === 'dark' ? theme.palette.primary.contrastText : theme.palette.primary.dark}>
-            <Typography variant="h5" align="left">
-              {spaceName}
-            </Typography>
-            <Typography variant="body2">{description}</Typography>
+        <Grid item xs={12} sm={12}>
+          <Box
+            display="flex"
+            flexDirection="column"
+            justifyContent="center"
+            alignItems="flex-start"
+            height={1}
+            width={1}
+            paddingLeft="1rem"
+            color={variant === 'dark' ? theme.palette.primary.contrastText : theme.palette.primary.dark}>
+            <Box>
```

Comment:
you don't really need a box here. This kind of makes the code hard to read and adds a ton of unnecessary inheritance. Better to just add a class to `Grid` and add styles there. This will definitely clean up your code imo